app/lib/medical-transcription-analysis-stack.ts

```diff
@@ -125,6 +125,17 @@ export class MedicalTranscriptionAnalysisStack extends cdk.Stack {
         });
 
         webAppS3Bucket.addToResourcePolicy(cloudfrontPolicyStatement);
+
+        // const cloudfrontStorageBucketPolicyStatement = new iam.PolicyStatement({
```

Comment:
Is there a reason we have this commented code or can we remove it?

Reply:
Removed.